### PR TITLE
Testing Pizzas Homework

### PR DESCRIPTION
**Pull request by Kasperice**

### CODE REVIEW

File: src/Pizzeria.cpp [hunk 1]
```diff
@@ -1,73 +1,53 @@
...
+    auto order = std::find_if(orders_.begin(), orders_.end(),
+                              [orderId](const auto& order) { return std::get<0>(order) == orderId; });
+    if (order != orders_.end()) {
         auto pizzas = std::get<Pizzas>(*order);
-        for (const auto & pizza : pizzas)
-        {
+        for (const auto& pizza : pizzas) {
             std::cout << "Baking " << pizza->getName() << std::endl;
-            std::this_thread::sleep_for(pizza->getBakingTime());
+            // std::this_thread::sleep_for(pizza->getBakingTime());
```
Comment (WBigaj) [inline src/Pizzeria.cpp:35]: Is this comment necessary?

**Review by WBigaj — COMMENTED** [review]
I have checked your code and have one question.

**Comment by jakjan95** [comment]
Looks Good :)

File: test/PizzeriaTest.cpp [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+
 #include <string>
-#include "mocks/PizzaMock.hpp"
-#include "Pizzeria.hpp"
-#include "Margherita.hpp"
+
 #include "Funghi.hpp"
+#include "Margherita.hpp"
+#include "Pizzeria.hpp"
+#include "mocks/DummyTimer.hpp"
```
Comment (jakjan95) [inline test/PizzeriaTest.cpp:8]: Since we dont use DummyTimer you dont need to include it :)

**Review by jakjan95 — APPROVED** [review]
Code looks good! :)

File: test/PizzeriaTest.cpp (hunk 2 above)
Comment (Kasperice) [reply test/PizzeriaTest.cpp:8]: Right, I forgot about it, thanks :) I will change it some time ;)

File: src/Pizzeria.cpp (hunk 1 above)
Comment (Kasperice) [reply src/Pizzeria.cpp:35]: It is not, I simply forgot to delete

File: test/PizzeriaTest.cpp [hunk 3]
```diff
@@ -28,40 +30,68 @@ TEST_F(PizzeriaTest, priceForMargherita25AndFunghi30ShouldBe55)
...
+
+    // When
+    auto orderId = pizzeria.makeOrder(pizzas);
+    pizzeria.bakePizzas(orderId);
+    pizzeria.completeOrder(orderId);
+    auto price = pizzeria.calculatePrice(orderId);
+
+    // Then
+    ASSERT_EQ(52, price);
 }
```
Comment (ziobron) [inline test/PizzeriaTest.cpp:97]: Please avoid magic values in your code. Use constexpr for values like minute(1), 22.0, minute(3), 20.0, etc.
Use them in proper places - EXPECT_CALL and ASSERT_EQ. It shows the connection between elements.

**Review by ziobron — APPROVED** [review]

**Comment by ziobron** [comment]
{ "requirement_ids" : [1119, 1119, 1119, 3524, 3525, 3526] }

3 * 3 XP za punktualność (1119)
10 XP za TimeMock w Pizzas (3524)
15 XP za nowy przypadek testowy w Pizzas (3525)
5 XP za workflow w Pizzas (3526)

**Pull request closed without merging**